Proposals/2024-04-05_LexPlay Hackathon (CaptureFlag).md

```diff
@@ -2,6 +2,7 @@
 <!-- as per gitbook [page](https://gitbook.lexdao.net/untitled/grants-committee-charter/grant-proposals) -->
 
 <!-- these are not necessarily exclusive -->
+
 [x] Reimburse (2x💵150/hr to [wallet](https://etherscan.io/address/0x590D24003D5Ec516502db08E01421ba56a5cd611 
 ) $150+250/hr @dclark) - for USPTO [wordmark](https://discord.com/channels/682960432272506907/1223273416690237551) extension+analysis [^1]
```

Comment:
The verbal agreement was $150 (fee) and $150/hr for dclark ... note for me I accept same though note the invoice will probably show a hefty discount to the actual hours worked ... this stuff is ... hard cognitive yakka